rust/gui-client/src/settings.ts

```diff
@@ -168,7 +171,23 @@ logsTabBtn.addEventListener("click", (_e) => {
 listen<Settings>('settings_changed', (e) => {
   let settings = e.payload;
 
-  authBaseUrlInput.value = settings.auth_base_url;
+  authBaseUrlInput.value = settings.auth_url;
   apiUrlInput.value = settings.api_url;
   logFilterInput.value = settings.log_filter;
+
+  authBaseUrlInput.disabled = settings.auth_url_is_managed
+  apiUrlInput.disabled = settings.api_url_is_managed;
+  logFilterInput.disabled = settings.log_filter_is_managed;
+
+  if (settings.auth_url_is_managed) {
+    authBaseUrlInput.dataset['tip'] = "This settings is managed by your organisation."
+  }
+
+  if (settings.api_url_is_managed) {
+    apiUrlInput.dataset['tip'] = "This settings is managed by your organisation."
+  }
+
+  if (settings.log_filter_is_managed) {
+    logFilterInput.dataset['tip'] = "This settings is managed by your organisation."
+  }
```

Comment:
Unfortunately, the Edge WebView doesn't appear to show this :(
On Linux, this works fine.

Reply:
I'll treat this as a known issue for now, we can fix it later. The inputs are disabled and greyed-out so the user can't change anything.

Reply:
Perhaps it is also an issue of my VM? @jamilbk Could you check this on your local Windows machine?